extra-multimedia/makemkv/autobuild/defines

```diff
@@ -0,0 +1,10 @@
+PKGNAME=makemkv
+PKGSEC="non-free/video"
+PKGDES="A converter to convert video clips from proprietary discs into a set of MKV files"
+PKGDEP="zlib qt-5 mesa openssl ffmpeg expat"
+
+# License issues
+ABSTRIP=0
+ABSPLITDBG=0
```

Comment:
Add an empty line below to denote that the following two lines are not related to licensing issue.